### ISSUE TITLE
Fail to load javascript when expanding topic message data

### ISSUE THREAD

**Issue by Adman** [tchiotludo/akhq#1501]
Hello,

After clicking on topic message data to expand it, I'm getting the following error message in developer console:
```
Refused to execute script from 'https://<my-domain>/kafka/ui/ui/33be477618f676447f25a3fa5252f3d0.js' because its MIME type ('text/html') is not executable.
```
The `context-path` is set to `"/kafka"`.
Most probably, the problem is that it's adding one more "ui" string into the url. If I remove one of the "ui" from the URL, it works.

**Comment by napalmCore** [comment]
Hi, i would like to help, could you assign me this?

**Comment by napalmCore** [comment]
Hi.
I was not able to reproduce the bug, i cloned the project and run the docker-compose-dev the project is built and run successfully,
When I go to the topic page and open the data tab I don't see any errors except a last pass extension related error.
I tested it on chrome 131.0.6778.265 and Edge 131.0.2903.112
See the screenshot attached
Could you please provide more context on how to reproduce this bug
![topics-data-ui-no-errors](https://github.com/user-attachments/assets/809db998-ec29-4b35-a3af-fba96db29549)

**Comment by Adman** [comment]
@napalmCore Try sending a json message to the topic and then expand it.

**Comment by napalmCore** [comment]
@Adman do you mean add JSON to the value field in create topic form? 
![add-topic](https://github.com/user-attachments/assets/ba12cece-54d0-4bc3-9488-6b3074c5030b)

**Comment by Adman** [comment]
Yes. Or create for example python client which is able to send that message there.

**Comment by napalmCore** [comment]
@Adman unfortunately i tried both methods and it works fine, i was not able to reproduce the bug. i tried even with a big JSON with nested data.

**Comment by hitdepani** [comment]
I WOULD LIKE TO WORK ON THIS
